### ISSUE TITLE
Moving out of persona authorization

### ISSUE THREAD

**Issue by ksripathi** [vlead/outreach-portal#208]
As persona is going to shutdown their services on 30 Nov 2016 authorization for Outreach Portal is no where available.

It was decided that we need move to the Google Oauth autherization.

**Comment by ksripathi** [comment]
We have two domains pointing to the same IP.

    domain1 : outreach-demo.vlabs.ac.in
    domain2 : outreach-demo.virtual-labs.ac.in

    domain1 has alias to domain2 

I have registered both the below api's as redirected apis with google dashboard as follows

    api1 : outreach-demo.vlabs.ac.in/authorized

    api2 : outreach-demo.virtual-labs.ac.in/authorized

I have integrated Google oauth service with outreach application and it is working as expected when i access using domain name "outreach-demo.virtual-labs.ac.in". with this I could login/logout and everything working perfect.

But,When I access application using domain `outreach-demo.vlabs.ac.in` and tried to login I got error saying **invalid response from google** by redirecting to 
`outreach-demo.virtual-labs.ac.in/authorized?code=4/7jglT8deFsB4gubhglR9XeGYpsovgciTy7xrrTpGwu8#`

I do not understand that even if i access my application using "`outreach-demo.vlabs.ac.in`" it is redirecting to "`outreach-demo.virtual-labs.ac.in/authorizedauthorized?code=47jglT8deFsB4gubhglR9XeGYpsovgciTy7xrrTpGwu8#`" by giving error **invalid response from google**
